client/X11/xf_cliprdr.c

```diff
@@ -908,11 +908,12 @@ static BOOL xf_cliprdr_process_property_notify(xfClipboard* clipboard,
         XEvent* xevent)
 {
 	xfCliprdrFormat* format;
-	xfContext* xfc = clipboard->xfc;
 
 	if (!clipboard)
 		return TRUE;
 
+	xfContext* xfc = clipboard->xfc;
+
```

Comment:
That will break older C compilers which require variables to be declared on top of a scope.